### PR TITLE
Change "Disciple Tools" to "Disciple.Tools". Fix duplicate menu

### PR DESCRIPTION
**Pull request by corsacca**

### CODE REVIEW

File: admin/admin-menu-and-tabs.php [hunk 1]
```diff
@@ -59,12 +59,11 @@ public function __construct() {
      * @since 0.1
      */
     public function register_menu() {
-        add_menu_page( __( 'Extensions (DT)', 'disciple_tools' ), __( 'Extensions (DT)', 'disciple_tools' ), 'manage_dt', 'dt_extensions', [ $this, 'extensions_menu' ], 'dashicons-admin-generic', 59 );
```
Comment (prykon) [inline admin/admin-menu-and-tabs.php:62]: Why was this removed?

Comment (corsacca) [reply admin/admin-menu-and-tabs.php:62]: @prykon Because the theme already declares the menu. And since this version is slightly different than the theme version it results in a duplicate declaration.

**Review by prykon — APPROVED** [review]
